### PR TITLE
Add map layer visibility and zoom to layer to step2 layer tabs

### PR DESCRIPTION
**Pull request by pmbohm**
Layer visibility functionality was removed by #1804 
The zoom to layer functionality was left out of the step2 accordion changes #1774

### CODE REVIEW

**Comment by pmbohm** [comment]
fixes #1798

File: web-app/js/portal/details/NCWMSColourScalePanel.js [hunk 1]
```diff
@@ -39,6 +39,9 @@ Portal.details.NCWMSColourScalePanel = Ext.extend(Ext.Panel, {
         });
 
         this.items = [
+            new Ext.Spacer({
+                height: 10
+            }),
```
Comment (dnahodil) [inline web-app/js/portal/details/NCWMSColourScalePanel.js]: I don't think it makes sense to add a spacer to this component. If we want there to be more space around it then that space should really be in the parent's list of items (ie. https://github.com/aodn/aodn-portal/pull/1807/files#diff-8da76661a875615270e9fb526f5efac0R86).

File: web-app/js/portal/lang/en.js [hunk 2]
```diff
@@ -160,13 +160,13 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
...
     autozoom: 'Auto zoom to data collection extent',
-    hideDetailsPanel: 'Hide data collection details',
+    showMapLayer: 'Shown on map',
     clearAllButtonLabel: '<span class="fa fa-fw fa-undo"></span> Clear and Reset',
     clearAllButtonTooltip: 'Remove all data collections from the Subset and Download pages, reset the map location and zoom level',
     removeButtonTooltip: 'Remove this data collection from the Subset and Download pages, the collection can be re-added from the search.',
 
     // FilterGroupPanel.js
-    clearFilterButtonLabel: '<span class="fa fa-fw fa-undo"></span> Clear subset',
+    clearFilterButtonLabel: '<span class="fa fa-search-plus"></span> Clear subset',
```
Comment (dnahodil) [inline web-app/js/portal/lang/en.js]: This looks like a mistake.

**Comment by dnahodil** [comment]
Looks good, @pmbohm and seems to work well. Just two comments.

File: web-app/js/portal/details/NCWMSColourScalePanel.js (hunk 1 above)
Comment (pmbohm) [inline web-app/js/portal/details/NCWMSColourScalePanel.js]: I went out of my way to specifically place it here. if there will be some HTML from the NCWMSColourScalePanel then i want the space. @dnahodil

**Pull request merged**